.circleci/config.yml

```diff
@@ -18,7 +18,6 @@ executors:
       AWS_REGION: us-east-1
```

Comment:
I know this PR is just a POC atm, so just a note to remove this file once we merge.

Reply:
I would need an admin to do a few things as well, also the ci hasn't been entirely transferred so somethings are still present in circle